tests/unit/test_federation_policy

```diff
@@ -34,7 +114,8 @@ cat > "$CONFIG_JSON" <<EOF
   "remote_hubs": []
 }
 EOF
-cf-agent --bundlesequence cfengine_enterprise_federation:config --no-lock --file ../../cfe_internal/enterprise/federation/federation.cf  --show-evaluated-classes | grep cfengine_enterprise_federation > output
+cf-agent --bundlesequence cfengine_enterprise_federation:config --no-lock --file ../../cfe_internal/enterprise/federation/federation.cf  --show-evaluated-classes 2>&1 | tee "$TMPLOG"
```

Comment:
nice

Reply:
Add `set -o pipefail` after `set -e` near the beginning of this file to make sure the test fails if the `cf-agent` run fails here (otherwise it will take the exit code of `tee`).